platform/ext/target/nordic_nrf/common/core/nrf_exception_info.c

```diff
@@ -11,34 +11,60 @@
 
 static struct nrf_exception_info nrf_exc_info;
 
-static void spu_dump_context(struct nrf_exception_info *ctx)
+static void dump_exception_info(struct nrf_exception_info *ctx)
 {
     SPMLOG_ERRMSG("Platform Exception: SPU Fault\r\n");
```

Comment:
Very nit but this should write something like more generic now that MPC maybe caused the fault?